css/gallery.css

```diff
@@ -31,12 +37,12 @@
   background-position: center;
   background-size: cover;
   position: relative;
-  width: 227px;
-  height: 227px;
   margin-right: 17px;
   margin-bottom: 17px;
   box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.75);
   overflow: hidden;
+  flex-basis: 20%;
```

Comment:
its not for you to change it, but why 20%?

Reply:
so that the size of the container between the elements is resolved by 20% and there is 4 in line

Reply:
but, 4 x 20% would be 80%, where would the other 20% be?